app/controllers/custom_fields_controller.rb

```diff
@@ -36,7 +37,12 @@ class CustomFieldsController < ApplicationController
 
   def index
     # loading wp cfs exclicity to allow for eager loading
-    @custom_fields_by_type = CustomField.all.where.not(type: 'WorkPackageCustomField').group_by { |f| f.class.name }
+    @custom_fields_by_type = CustomField.all
+      .where.not(type: 'WorkPackageCustomField')
+      # ProjectCustomFields now managed in a different UI
+      .tap { |query| query.where.not(type: 'ProjectCustomField') }
```

Comment:
Just a nitpick, type can also take an array of strings
```suggestion
      .where.not(type: ['WorkPackageCustomField', 'ProjectCustomField'])
```